translations/ui-myprofile/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "meta.title": "My profile",
   "settings.index.paneTitle": "My profile",
-  "settings.changePassword": "Change password"
+  "settings.changePassword": "Change password",
+  "caption": "Change Password"
```

Comment:
I think it should be like settings.changePassword.caption or else

Reply:
changed